x-pack/plugins/apm/public/components/shared/slo_callout/index.tsx

```diff
@@ -76,12 +82,7 @@ export function SloCallout({
         <EuiFlexItem grow={false}>
           <EuiFlexGroup>
             <EuiFlexItem>
-              <EuiLink
-                data-test-subj="apmCreateSloLink"
-                href={basePath.prepend(
-                  `/app/observability/slos/create${sloParams}`
-                )}
-              >
+              <EuiLink data-test-subj="apmCreateSloLink" onClick={handleClick}>
```

Comment:
I added the `EuiButton` to the `EuiLink` because it didn't accept both `href` and `onClick`, with this change we can get rid of the `EuiButton` below, and do the `handleClick` and `dismissCallout` both on the `onClick` on EuiLink

```
<EuiLink 
data-test-subj="apmCreateSloLink" 
onClick={ ()=> {
        handleClick()
        dismissCallout()
}}>
```

Reply:
good catch. Will update it